### PR TITLE
adding health warning about insecure protocols for LOAD CSV

### PR DESCRIPTION
**Pull request by phil198**
adds a warning against allowing insecure protocols for `LOAD CSV`

this PR is the docs counterpart to https://github.com/neo-technology/neo4j/pull/26403

I will add a duplicate warning to  
https://neo4j.com/docs/cypher-manual/current/clauses/load-csv/
https://neo4j.com/docs/operations-manual/5/authentication-authorization/load-privileges/ 
and possibly
https://neo4j.com/docs/getting-started/data-import/csv-import/

once the wording of this one is finalised.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1006.surge.sh

Updated pages:
- [LOAD CSV](https://neo4j-docs-cypher-1006.surge.sh/cypher-manual/5/clauses/load-csv)

**Review by OliviaYtterbrink — APPROVED** [review]
The information is good, I'll leave the wording to the docs team. 

(missing the `is` after the first `It`)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1006.surge.sh

Updated pages:
- [LOAD CSV](https://neo4j-docs-cypher-1006.surge.sh/cypher-manual/5/clauses/load-csv)

File: modules/ROOT/pages/clauses/load-csv.adoc [hunk 1]
```diff
@@ -90,6 +90,12 @@ You can import data from a CSV file hosted on a remote path.
...
 
+[IMPORTANT]
+====
+It is strongly recommended to only allow secure protocols like HTTPS in favour of allowing insecure protocols like HTTP. If allowing an insecure protocol is completely unavoidable, then Neo4j internally takes some measures to make these requests as secure as possible within their limitations. However, this means that insecure URLs which are located somewhere which uses virtual hosting will not work. The only way to load an insecure resource from a virtually hosted URL is to add the JVM argument `-Dsun.net.http.allowRestrictedHeaders=true` to the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings/#config_server.jvm.additional/[jvm.additional]  config setting. For the sake of security, it is strongly recommended that this be avoided at all cost and that you only permit loading of resources over secure protocols. This can be achieved by limiting link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/load-privileges/#access-control-load-cidr/[load privileges] to trusted sources that use secure protocols.
```
Comment (renetapopova) [inline modules/ROOT/pages/clauses/load-csv.adoc]:
```suggestion
It is strongly recommended to permit resource loading only over secure protocols such as HTTPS instead of insecure protocols like HTTP by limiting the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/load-privileges/#access-control-load-cidr/[load privileges] to trusted sources that use such protocols and avoid using resources from untrusted sources. 
If allowing an insecure protocol is absolutely unavoidable, Neo4j has internal measures to enhance the security of these requests within their limitations. 
However, this means that insecure URLs on virtual hosts will not function unless you add the JVM argument `-Dsun.net.http.allowRestrictedHeaders=true` to the configuration setting link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings/#config_server.jvm.additional/[jvm.additional]. 
```

Comment (renetapopova) [reply modules/ROOT/pages/clauses/load-csv.adoc]: How about this wording?

**Review by renetapopova — COMMENTED** [review]
Thanks, @phil198. I added an editorial suggestion.

File: modules/ROOT/pages/clauses/load-csv.adoc (hunk 1 above)
Comment (phil198) [inline modules/ROOT/pages/clauses/load-csv.adoc]: Thanks. That looks better. Just a couple of suggestions:

```suggestion
It is strongly recommended to permit resource loading only over secure protocols such as HTTPS instead of insecure protocols like HTTP. 
This can be done by limiting the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/load-privileges/#access-control-load-cidr/[load privileges] to only trusted sources that use secure protocols. 
If allowing an insecure protocol is absolutely unavoidable, Neo4j takes measures internally to enhance the security of these requests within their limitations. 
However, this means that insecure URLs on virtual hosts will not function unless you add the JVM argument `-Dsun.net.http.allowRestrictedHeaders=true` to the configuration setting link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings/#config_server.jvm.additional/[jvm.additional].
```

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.
